cds_migrator_kit/videos/weblecture_migration/load/load.py

```diff
@@ -27,9 +50,162 @@ def _prepare(self, entry):
         """Prepare the record."""
         pass
 
+    def create_publish_single_video_record(self, metadata):
+        """Create and publish project and video for single main video record."""
+        # Create project
+        project_metadata = {"category": "CERN", "type": "VIDEO"}
+        project_deposit = Project.create(project_metadata)
+        # Create video
+        metadata["_project_id"] = project_deposit["_deposit"]["id"]
+        video_deposit = Video.create(metadata)
+        video_bucket = Bucket.get(video_deposit["_buckets"]["deposit"])
+        video_deposit_id = video_deposit["_deposit"]["id"]
+
+        # Upload video (main video)
+        bucket_id = video_deposit["_buckets"]["deposit"]
+        # dummy test video, files are not implemented yet
+        video_path = (
+            "cds_migrator_kit/videos/weblecture_migration/data/videos/test_video.mp4"
+        )
+        object_version = self._upload_video_file_to_bucket(
+            bucket_id=bucket_id, video_path=video_path
+        )
+
+        # Create tags, flow tasks and run
+        flow = self._run_flow(object_version, video_deposit_id)
+
+        # Publish video
+        self._publish_video_record(deposit_id=video_deposit_id)
+
+    def _run_flow(self, object_version, deposit_id):
+        """Low level represantation of post api/flows.
+
+        This methods creates updates ObjectVersionTag,
+        Creates the flow with tasks:
+        - ExtractMetadataTask
+        - ExtractFramesTask
+        and runs these tasks sync
+        """
+        deposit_id = str(deposit_id)
+        payload = dict(
+            version_id=str(object_version.version_id),
+            key=object_version.key,
+            bucket_id=str(object_version.bucket_id),
+            deposit_id=str(deposit_id),
+        )
+        # Create FlowMetadata
+        flow = FlowMetadata.create(
+            deposit_id=deposit_id,
+            user_id=2,  # TODO It's failing vithout user id
```

Comment:
Comment: I think this would be nice to discuss together (with Jose and Nico), how should we tackle this, should we rely on actual user that uploaded the record originally if they do exist? or do we use concrete "system" user for all this operations?